src/main/kotlin/me/serce/solidity/ide/errors.kt

```diff
@@ -37,10 +37,7 @@ class SentryReportSubmitter : ErrorReportSubmitter() {
     parentComponent: Component,
     consumer: Consumer<SubmittedReportInfo>
   ): Boolean {
-    val ijEvent = events.firstOrNull()
-    if (ijEvent == null) {
-      return true
-    }
+    val ijEvent = events.firstOrNull() ?: return true
```

Comment:
I intentionally try to avoid `nullable ?: return smth` idiom because it often hides the control flow in debugging, e.g. it's often unclear at which moment the method gets returned.

Reply:
Ok, reverted.